### DISCUSSION TITLE
Support for URLs to open organic maps with a route pre populated

### DISCUSSION THREAD

**Discussion by ivaanmehrotra** [organicmaps/organicmaps#1978] [Ideas]
Hi,

I will like to propose a feature similar to https://developers.google.com/maps/documentation/urls/get-started#directions-action where on accessing a URL, the organic maps app will open with the route prepopulated and ready to navigate.

**Comment by ivaanmehrotra** [comment]
I think this idea has already been implemented as the following URL opens the route

mapswithme://route?sll=50.183933,8.942871&saddr=Start%20Point&dll=49.998912,8.278198&daddr=EndPoint&type=vehicle and it worked well and opened up organicmaps with the route prepopulated and ready to start. However, if I replace mapswithme:// with om:// then it opens the app but the route is not shown so it might be a small bug.

**Reply by biodranik** [reply]
In the next update om:// also should work
